.eslintrc

```diff
@@ -1,2 +1,4 @@
 # We use JavaScript Standard Style (https://github.com/feross/standard)
 extends: standard
+rules:
```

Comment:
This PR is for `NotificationScheduler`, not for fixing 143 eslint errors spread across the entire project.